packages/hidp_wagtail/hidp_wagtail/checks.py

```diff
@@ -22,3 +22,26 @@ def check_wagtailadmin_login_url(app_configs, **kwargs):
             )
         )
     return errors
+
+
+@register(Tags.settings)
+def check_account_management_links_context_processor(app_configs, **kwargs):
+    """
+    Custom system check to ensure that the 'account_management_links' context processor is configured.
+
+    This check verifies that the 'account_management_links' context processor is present in the Django settings.
+    If the context processor is not found, it raises an error with a hint on how to set it.
+    """
+    errors = []
+    if (
+        "hidp_wagtail.context_processors.account_management_links"
+        not in settings.TEMPLATES[0]["OPTIONS"]["context_processors"]
+    ):
```

Comment:
For the rare case that a user has multiple template engines, should we support the case where the one which has this configuration is not the first engine?
If that were the case, would it be ok if this context processor is configured for at least one of them?

```suggestion
    if all(
        "hidp_wagtail.context_processors.account_management_links"
        not in template_engine["OPTIONS"]["context_processors"]
        for template_engine in settings.TEMPLATES
    ):
```

Reply:
Done!